apps/accelerate/open_alpha_tensor/open_alpha_tensor/core/actors/stage.py

```diff
@@ -274,19 +306,14 @@ def monte_carlo_tree_search(
     state_hash = to_hash(extract_present_state(state))
     if state_hash in state_dict:
         with torch.no_grad():
-            N_s_a = state_dict[state_hash][3]
+            N_s_a = state_dict[state_hash][2]
             n_sim -= int(N_s_a.sum())
             n_sim = max(n_sim, 0)
 
     for _ in range(n_sim):
         simulate_game(model, state, t_time, n_steps, game_tree, state_dict)
     # return next state
-    (state_tensor, actions), _, repetitions, N_s_a, q_values, _ = state_dict[
-        state_hash
-    ]
-    possible_states = extract_children_states_from_actions(
-        state_tensor, actions
-    )[0]
+    possible_states, repetitions, N_s_a, q_values, _ = state_dict[state_hash]
```

Comment:
please keep the old code modification (the code in red should replace the code in green)